docs/devenv_rhel8.md

```diff
@@ -49,9 +51,9 @@ Log into the virtual machine using SSH with the `microshift` user credentials.
 
 Run the following commands to configure SUDO, upgrade the system, install basic dependencies and enable remote Cockpit console.
 ```bash
-sudo echo -e 'microshift\tALL=(ALL)\tNOPASSWD: ALL' > /etc/sudoers.d/microshift
+echo -e 'microshift\tALL=(ALL)\tNOPASSWD: ALL' | sudo tee /etc/sudoers.d/microshift
 sudo dnf update -y
-sudo dnf install -y git cockpit make golang selinux-policy-devel rpm-build
+sudo dnf install -y git cockpit make golang selinux-policy-devel rpm-build bash-completion
```

Comment:
if the hypervisor host does not have `virt-install` installed, the `create-vm.sh` script will complain. Should we include it?

Reply:
And apparently `virt-viewer`